package.json

```diff
@@ -30,31 +29,12 @@
   },
   "type": "commonjs",
   "main": "dist/index.js",
-  "module": "dist/esm/index.mjs",
-  "types": "dist/index.d.ts",
+  "module": "dist/index.mjs",
   "exports": {
-    "./dist/*": {
```

Comment:
**issue (blocking):** v7 of node-pg-migrate is currently serving `/dist` as a backwards compatibility to what was possible in v6.2.2  
I cant remove this without a major release